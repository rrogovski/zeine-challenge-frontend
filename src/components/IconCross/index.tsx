import * as S from './styles'

const IconCross = () => <S.Wrapper />

export default IconCross
