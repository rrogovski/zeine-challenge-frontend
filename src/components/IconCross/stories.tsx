import { Meta, StoryObj } from '@storybook/react'
import IconCross from '.'

export default {
  title: 'IconCross',
  component: IconCross
} as Meta

export const Default: StoryObj = {}
