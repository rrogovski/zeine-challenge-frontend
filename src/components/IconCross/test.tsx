import { render } from '@testing-library/react'

import IconCross from '.'

describe('<IconCross />', () => {
  it('should render the heading', () => {
    const { container } = render(<IconCross />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
