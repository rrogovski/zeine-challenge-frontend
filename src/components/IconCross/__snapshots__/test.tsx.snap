// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconCross /> should render the heading 1`] = `
<img
  class=""
/>
`;
